stable/opa/templates/webhookconfiguration.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
 {{ include "opa.labels.standard" . | indent 4 }}
 webhooks:
   - name: webhook.openpolicyagent.org
+    namespaceSelector:
+      matchLabels:
+        openpolicyagent.org/policy: rego
```

Comment:
@cplee to prevent self-management, I think what we want is something more like this:

```
namespaceSelector:
  matchExpressions:
  - {key: openpolicyagent.org/webhook, operator: NotIn, values: [ignore]}
```

Then the namespace that OPA is deployed into must be labelled with:

```
openpolicyagent.org/webhook: ignore
```

It may also be a good idea to exclude kube-system as well to prevent the cluster from being able to heal itself.

As is there are a couple problems with this change:

1. `openpolicyagent.org/policy: rego` is used for labeling configmaps for kube-mgmt. Let's use something else
2. by using `matchLabels` it means that all namespaces must be labeled which is a bit invasive and also error prone (e.g., if you forget to label, then your admission policies aren't getting enforced)
3. a bit less of an issue, since preventing self-management is probably better, but this wouldn't be backwards compatible so we should make it off-by-default for now and then in future we can flip it.

Reply:
Thanks @tsandall, I've pushed a new commit that takes your feedback into consideration.  That was my bad, I misread [the docs](https://www.openpolicyagent.org/docs/v0.10.7/kubernetes-admission-control/#4-define-a-policy-and-load-it-into-opa-via-kubernetes) and incorrectly assumed that OPA already required namespaces to opt-in with that label...I know realize it is just the configmap that needs the label.